src/Config.php

```diff
@@ -187,21 +187,15 @@ private function setSender($c)
             $default = "Rollbar\Senders\FluentSender";
 
             if (isset($c['fluent_host'])) {
-                $c['senderOptions'] = array(
-                    'fluentHost' => $c['fluent_host']
-                );
+                $c['senderOptions']['fluentHost'] = $c['fluent_host'];
             }
 
             if (isset($c['fluent_port'])) {
-                $c['senderOptions'] = array(
-                    'fluentPort' => $c['fluent_port']
-                );
+                $c['senderOptions']['fluentPort'] = $c['fluent_port'];
             }
 
             if (isset($c['fluent_tag'])) {
-                $c['senderOptions'] = array(
-                    'fluentTag' => $c['fluent_tag']
-                );
+                $c['senderOptions']['fluentTag'] = $c['fluent_tag'];
```

Comment:
So, what I mean is that:

```
$c['senderOptions'] = array(
     'fluentTag' => $c['fluent_tag']
);
```
Would override the entire sender config and the last setting wins. If you set more than one additional thing like `host` and `tag` the sender config would contain only `tag`.

While 
```
$c['senderOptions']['fluentTag'] = $c['fluent_tag'];
```
would extend the sender options by whatever is set on the right side of the `=`.

I hope this makes it clearer. :)